var/spack/repos/builtin/packages/libstdcompat/package.py

```diff
@@ -110,5 +110,6 @@ def cmake_args(self):
 
     @run_after("build")
     @on_package_attributes(run_tests=True)
-    def test(self):
+    def test_execute(self):
```

Comment:
This is a build-time, not stand-alone, test method (https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-installation-phase-tests).  So the method name should not be starting with `test_`.

Reply:
🤔 I am surprised by the output given the use of the directives.

Reply:
And where is it running when it runs as a stand-alone test?

Reply:
And why is it reporting success/passing when the software is built *without* `--test=root`.

Reply:
There is NOTHING in the test stage or relevant in the prefix for this to work as a stand-alone test.

The fact that Spack is acting as though it did something in the stand-alone test is troublesome.

Reply:
And a bug in spack.

Reply:
Please change this so it is ONLY a build-time test.

Reply:
Nvm.  The method should just be removed.